dozer/cogs/roles.py

```diff
@@ -357,16 +369,18 @@ def from_role(cls, role):
 
 
 class MissingMember(db.DatabaseObject):
+    """Is this necessary any more?"""
```

Comment:
Yes, this is required for the relationship with the MissingRole class and table.